### PR TITLE
Dropseq tools

### PR DESCRIPTION
**Pull request by cgirardot**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by cgirardot** [comment]
@bioconda/core this PR adds a new package. This is Drop-seq_tools suite from the Mc Carroll lab http://mccarrolllab.com/dropseq/. The suite basically ships as a jar and shell wrappers for each sub-command. The recipe is very simple and is basically an adaptation of the Je-suite one already in bioconda. I could run CircleCI locally successfully.

File: recipes/Drop-seq_tools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{% set name = "Drop-seq_tools" %}
```
Comment (bgruening) [inline recipes/Drop-seq_tools/meta.yaml]: Can we have small letters here?

File: recipes/Drop-seq_tools/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  license: MIT License
+  license_family: MIT
+  summary: |
+    Package fot the analysis of Drop-seq data developed by Jim Nemesh in the McCarroll Lab
+package:
+  name: dropseq_tools
+  version: {{ version }}
+
+build:
+  number: 1
```
Comment (bgruening) [inline recipes/Drop-seq_tools/meta.yaml]: Please start with 0.

File: recipes/Drop-seq_tools/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  url: http://mccarrolllab.com/download/1276/{{ name }}-{{ version }}.zip
+  md5: 28533edd82867ed75867b05785cf8834
+
+requirements:
+  run:
+    - openjdk >=7
+
+test:
+  commands:
+    - '[[ $(TrimStartingSequence -- --version 2>&1) == "1.13(7bed8f4_1513008033)" ]]'
```
Comment (bgruening) [inline recipes/Drop-seq_tools/meta.yaml]: Include a few more tests e.g. the --help from the nodes?

File: recipes/Drop-seq_tools/meta.yaml (hunk 1 above)
Comment (cgirardot) [reply recipes/Drop-seq_tools/meta.yaml]: the name is used to build the file name to download from their web site. I'd rather keep it like this if possible.

File: recipes/Drop-seq_tools/meta.yaml (hunk 2 above)
Comment (cgirardot) [reply recipes/Drop-seq_tools/meta.yaml]: sure

File: recipes/Drop-seq_tools/meta.yaml (hunk 3 above)
Comment (cgirardot) [reply recipes/Drop-seq_tools/meta.yaml]: ok can do this, give a sec

**Comment by cgirardot** [comment]
@bgruening , just pushed a new version in which : 
- I added a test on every sub-command of the package
- build number is 0
- kept the upper case in name though, lemme know if thats OK

**Comment by cgirardot** [comment]
@bgruening I am not sure if I should do something for this macOS plan ?

**Pull request merged**